spec/test_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../lib/board'
+
+describe Board do
+  
+  let(:board) { Board.new }
+  describe 'update' do
+    it 'update the move' do
+      expect(board.update(2, 'X')).to eql('X')
+    end
+  end
+  describe 'update' do
+    it 'update the move' do
+      expect(board.update(1, 'O')).to eql('O')
+    end
+  end
+  describe 'winner?' do
+    it 'checks the winner' do
+      expect(board.winner?).to eql(false)
+    end
+  end
+
+  describe 'valid_move' do
```

Comment:
You made a good test here, but try to create another test if a player makes an invalid move, in this way your tests will be more complete 🥇